src/compiler/types.ts

```diff
@@ -892,41 +892,33 @@ module ts {
         referencedFiles: FileReference[];
 
         hasNoDefaultLib: boolean;
-        externalModuleIndicator: Node; // The first node that causes this file to be an external module
+
+        // The first node that causes this file to be an external module
+        externalModuleIndicator: Node;
         languageVersion: ScriptTarget;
         identifiers: Map<string>;
         
-        // @internal
-        nodeCount: number;
-
-        // @internal
-        identifierCount: number;
-
-        // @internal
-        symbolCount: number;
+        /* @internal */ nodeCount: number;
+        /* @internal */ identifierCount: number;
+        /* @internal */ symbolCount: number;
 
-        // @internal
         // Diagnostics reported about the "///<reference" comments in the file.
-        referenceDiagnostics: Diagnostic[];
+        /* @internal */ referenceDiagnostics: Diagnostic[];
         
-        // @internal
         // Parse errors refer specifically to things the parser could not understand at all (like 
         // missing tokens, or tokens it didn't know how to deal with).
-        parseDiagnostics: Diagnostic[];
+        /* @internal */ parseDiagnostics: Diagnostic[];

```

Comment:
Is there a particular reason referenceDiagnostics and parseDiagnostics are separate?